### ISSUE TITLE
Failed do examples

### ISSUE THREAD

**Issue by iysheng** [snhobbs/kikit-multipanel#7]
* kikit-multipanel version:1.1.1
* Python version:3.11
* Operating System:fedora 38

### Description

when I do command with ``/usr/local/bin/kikit_multipanel  --fname panel.xlsx --out panel.kicad_pcb --preset_f preset.json`` but get these below errors:
``` bash
▸ /usr/local/bin/kikit_multipanel  --fname panel.xlsx --out panel.kicad_pcb --preset_f preset.json
Traceback (most recent call last):
  File "/usr/local/bin/kikit_multipanel", line 33, in <module>
    sys.exit(load_entry_point('kikit-multipanel==0.1.1', 'console_scripts', 'kikit_multipanel')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: main() missing 1 required positional argument: 'verison'
```

### What I Did

```
just want to test demo.

▸ kikit_multipanel --version
kikit_multipanel, version 0.1.1

```

**Comment by snhobbs** [comment]
Ah, yes. Fixed in V0.1.2.

**Comment by iysheng** [comment]
> Ah, yes. Fixed in V0.1.2.

:ok:
